framework/source/class/qx/test/ui/form/TextField.js

```diff
@@ -0,0 +1,58 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2011 1&1 Internet AG, Germany, http://www.1und1.de
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * Henner Kollmann
+
+************************************************************************ */
+
+qx.Class.define("qx.test.ui.form.TextField",
+{
+  extend: qx.test.ui.LayoutTestCase,
+
+  members:
+  {
+    setUp: function() {
+      this.__field = new qx.ui.form.TextField();
+      this.getRoot().add(this.__field);
+    },
+
+    tearDown: function() {
+      this.__field.destroy();
+      this.base(arguments);
```

Comment:
I can do this. 
In the other test the variable isnt't set to null. So i did not do this.